src/components/Subthoughts.native.tsx

```diff
@@ -107,6 +108,19 @@ const findFirstEnvContextWithZoom = (
   return child ? [...env[child.value], '=focus', 'Zoom'] : null
 }
 
+/** Get thought space. */
+const getThoughtSpacing = (distance: number, depth: number): number => {
+  if (distance === 0 || (distance === 1 && depth !== 0)) {
+    return 20
+  }
+
+  if (distance === 1 && depth === 0) {
+    return 5
+  }
+
+  return 0
```

Comment:
A ternary would be cleaner here. Single return statements are easier to reason about than multiple return statements. I try to only use additional return statements when short-circuiting from the beginning of a function.

Reply:
Code refactored.